.circleci/generate.sh

```diff
@@ -176,7 +180,7 @@ elif (!($has_env_vars)); then
 fi
 
 # add new or modified tests to the sets of tests to be repeated
-if (!($all)); then
+if (! ($all) && ! ($skip_changed_test_detection)); then
```

Comment:
This can be simplified by inverting the meaning of the var: https://github.com/adelapena/cassandra/commit/33ffb5fa30477317dabde9e64eea4f7a3cfe99a1